sdk/cmake/host-tools.cmake

```diff
@@ -35,11 +35,15 @@ function(configure_host_tools HOST_TOOLS_DIR)
         list(APPEND _target_locations ${_target_location})
     endforeach()
 
+    # When building for MSVC, inhibit the Release/Debug build currently selected in the ide
+    if (MSVC_IDE)
```

Comment:
Any reason we shouldn't just do this unconditionally?

Reply:
`$(ConfigurationName)` is an msvc specific macro

Reply:
I think you can just use a generator expression instead, e.g. `$<CONFIG>`

Reply:
Thanks, didnt know that!